src/core/reducers/api.js

```diff
@@ -70,6 +63,7 @@ export type ApiState = {
   clientApp: ?string,
   // See config.get('langs') for all possible values.
   lang: ?string,
+  regionCode: ?string,
```

Comment:
I am not sure but I think it should be `string | null` here (and probably the same for the other attributes). Not a big deal, though.

Reply:
That's true, given the values for `initialState`. I wonder why Flow isn't complaining about that? I guess maybe Flow allows null for optional properties (but I agree it shouldn't). We could either change the types here, or just make `initialState` have `undefined` instead of `null`. The latter is likely better because there isn't a way to set any of the values to `null`, other than in `initialState`.

Reply:
It turns out that [Maybe types can accept Null](https://flow.org/en/docs/types/maybe/), so this isn't actually incorrect, in terms of how Flow works anyway.

Reply:
I'm updating it anyway, so that initial values are set to `undefined`.

Reply:
Yes, I knew that but the maybe type is for "Typing values that may or may not exist". In this case, the value will always be present.

Reply:
In practice, yes, but theoretically no. If we don't get the header from the request, we don't set anything for `regionCode`.